### ISSUE TITLE
Compile fails when Verbose compilation is enabled

### ISSUE THREAD

**Issue by zayenz** [MiniZinc/MiniZincIDE#117]
Compiling in the MiniZinc IDE fails when Verbose compilation is enabled with an error "Could not determine model parameters", even if there are no parameters in the model.

Example model to test with:
```
var int: x;
var int: y;

constraint x < y;

solve maximize x + y;
```

**Comment by cyderize** [comment]
I think this should be fixed by https://github.com/MiniZinc/libminizinc/commit/ef300225e95ac24fbfde99ec5abf095a5b76cfe9 in the upcoming patch release.

**Comment by cyderize** [comment]
This has now been fixed in 2.5.1
